test/test-network/systemd-networkd-tests.py

```diff
@@ -2242,6 +2242,7 @@ def test_route_static(self):
 
         copy_unit_to_networkd_unit_path('25-address-static.network')
         check_output(*networkctl_cmd, 'reload', env=env)
+        time.sleep(1)
```

Comment:
Hmm, is `reload` asynchronous? I thought it does a (synchronous) bus call?

Reply:
Sorry, the explanation was incorrect. The operation reloading .network files are synchronous. But subsequent reconfiguration of network interfaces are asynchronous.